x-pack/plugins/observability_solution/apm/server/routes/services/get_service_metadata_details.ts

```diff
@@ -166,21 +162,26 @@ export async function getServiceMetadataDetails({
         },
         totalNumberInstances: { cardinality: { field: SERVICE_NODE_NAME } },
       },
+      fields: ['*'],
     },
   };
 
   const response = await apmEventClient.search('get_service_metadata_details', params);
 
-  const hit = response.hits.hits[0]?._source as ServiceMetadataDetailsRaw | undefined;
-  if (!hit) {
+  const fields = response.hits.hits[0]?.fields;
+  const fieldsNorm = (fields ? serviceMetadataDetailsMapping(fields) : undefined) as
+    | ServiceMetadataDetailsRaw
+    | undefined;
```

Comment:
We have a check for `undefined` in `serviceMetadataDetailsMapping` therefore we can pass `undefined` safely. No need to double-check. Wdyt? I can update it my PR if you agree

```suggestion
const fieldsNorm = serviceMetadataDetailsMapping(fields);
```